apps/emqx_auth/src/emqx_authz/sources/emqx_authz_client_info.erl

```diff
@@ -115,8 +115,9 @@ authorize(#{acl := Acl} = Client, PubSub, Topic, _Source) ->
         {error, MatchResult} ->
             MatchResult
     end;
+%% no acl in client info, so we skip this.
 authorize(_Client, _PubSub, _Topic, _Source) ->
-    nomatch.
+    skip.
```

Comment:
```suggestion
    ignore.
```
?

Reply:
ignore will print a ignore trace log on all acl hook run, which is useless.
so I add new type to skip this when no acl in client_info.
https://github.com/emqx/emqx/blob/master/apps/emqx_auth/src/emqx_authz/emqx_authz.erl#L466
This line always add a `client_info_source()` in the sources...

Reply:
https://github.com/emqx/emqx/blob/master/apps/emqx_auth/src/emqx_authz/emqx_authz.erl#L609-L610

which means authz **always** check client info and then run `authz_http`, `authz_mongo` and etc...
maybe we should keep `ignore` and make the `?TRACE` macro record it.

Reply:
IMO, we should not skip `ignore` trace. `ignore` means no rule found for the given client's current action, which is useful debug info e.g. when try to lookup mysql but no matching rows, we'd like to know this from the trace.